tests_recording/README.md

```diff
@@ -25,23 +31,28 @@ all credentials there eg:
 
 ### Command to run
 
-Ensure to remove files and directories for tests what you would like to
-regenerate.
+- Ensure to remove files and directories for tests what you would like to regenerate.
+- you have to have configs
+  - for packit: `~/.config/packit.yaml`
+  - for copr: `~/.config/copr`
 
 ```
-PAGURE_TOKEN=your_token GITHUB_TOKEN=your_token pytest-3 -v tests/testsuite_recording
+pytest-3 -v tests/testsuite_recording
 ```
 
+### Postprocessing
+
+- Remove secrets from stored files - Some parts are not covered by generic requre pre-commit hook for removing secrests, e.g. token and login for copr. Remove them manually or via requre-patch tools
+  ```
+  requre-patch purge --replaces "copr.v3.helpers:login:str:somelogin" --replaces "copr.v3.helpers:token:str:sometoken" tests_recording/test_data/*/*yaml
+  ```
+- create symlinks for files in tests_data, `file` handler stores data to tar archives and more tests uses same archives, so you can symlink them via requre tool:
+  ```
+  requre-patch create-symlinks `pwd`/tests_recording/test_data/
+  ```
+
 #### Matrix
 
 Install `requre` from git master branch
 
-- first round
-  - `ogr` - from master branch
-  - `rebasehelper` - version `0.19.0` and above
-- second round for test `for test_version_change_exception`
-  - `ogr` - from master branch
-  - `rebasehelper` - `0.17.1`
-- third round
-  - `ogr` - released (rpm or pypi)
-  - `rebasehelper` - `0.19.0` and above
+- In current version of packit there are no issues with various versions libraries. In case it happens, regenerate data with proper versions of libraries and add there keys for these versions.
```

Comment:
If I do this for the first time, I would not know what you mean by this. Maybe meantion what `key` are you talking about here.